flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/batch/connectors/cassandra/OutputFormatBaseTest.java

```diff
@@ -0,0 +1,317 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.batch.connectors.cassandra;
+
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.core.testutils.CheckedThread;
+import org.apache.flink.util.Preconditions;
+import org.apache.flink.util.concurrent.FutureUtils;
+
+import com.google.common.util.concurrent.ListenableFuture;
+import org.junit.jupiter.api.Test;
+
+import javax.annotation.Nullable;
+
+import java.io.IOException;
+import java.time.Duration;
+import java.util.LinkedList;
+import java.util.Queue;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Executor;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.function.Function;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+/** Tests for the {@link OutputFormatBase}. */
+public class OutputFormatBaseTest {
+
+    private static final Duration DEFAULT_MAX_CONCURRENT_REQUESTS_TIMEOUT =
+            Duration.ofMillis(Long.MAX_VALUE);
+
+    @Test
+    public void testSuccessfulWrite() throws Exception {
```

Comment:
```suggestion
class OutputFormatBaseTest {

    private static final Duration DEFAULT_MAX_CONCURRENT_REQUESTS_TIMEOUT =
            Duration.ofMillis(Long.MAX_VALUE);

    @Test
    void testSuccessfulWrite() throws Exception {
```
We can reduce the visibility because of junit5.

Reply:
I'll fix this during the merge.